extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/multipart/MultipartResponseTest.java

```diff
@@ -209,15 +223,24 @@ public MultipartData getFile() throws IOException {
         @Produces(MediaType.MULTIPART_FORM_DATA)
         @Path("/small")
         public MultipartData getSmallFile() throws IOException {
-            File file = File.createTempFile("toDownload", ".txt");
-            file.deleteOnExit();
+            File file = createTemporalFileToDownload();
             // let's write Woo hoo, woo hoo hoo 1 time
             try (FileOutputStream out = new FileOutputStream(file)) {
                 out.write(WOO_HOO_WOO_HOO_HOO.getBytes(StandardCharsets.UTF_8));
             }
             return new MultipartData("foo", file, null, 1984, null);
         }
 
+        @GET
+        @Produces(MediaType.MULTIPART_FORM_DATA)
+        @Path("/large")
+        public MultipartData getLargeFile() throws IOException {
+            File file = createTemporalFileToDownload();
+            RandomAccessFile f = new RandomAccessFile(file, "rw");
+            f.setLength(ONE_GIGA);
```

Comment:
I think we should also have verification if the file content is okay

Reply:
I don't see it (other than file existence)

Reply:
I've just updated the PR now.